Sources/Swift/Integrations/SessionReplay/SentryReplayOptions.swift

```diff
@@ -216,39 +278,108 @@ public class SentryReplayOptions: NSObject, SentryRedactOptions {
      *  - errorSampleRate Indicates the percentage in which a 30 seconds replay will be send with
      * error events.
      */
-    public init(sessionSampleRate: Float = 0, onErrorSampleRate: Float = 0, maskAllText: Bool = true, maskAllImages: Bool = true, enableViewRendererV2: Bool = false, enableFastViewRendering: Bool = false) {
+    public init(
+        sessionSampleRate: Float = 0,
+        onErrorSampleRate: Float = 0,
+        maskAllText: Bool = true,
+        maskAllImages: Bool = true,
+        enableViewRendererV2: Bool = true,
+        enableFastViewRendering: Bool = false
+    ) {
+        // Note for future maintainers:
+        // - This initializer is publicly available for Swift, but not for Objective-C.
+        // - Each parameter has a default value, so the parameter can be omitted.
+        // - The values are not optional, because SDK users should not be able to set them to nil.
+
+        // WHEN CHANGING ANY DEFAULT VALUE IN THE INITIALIZER, UPDATE THE PROPERTY DOCUMENTATION
```

Comment:
`l`: Maybe a bit more specific.
```suggestion
        // WHEN CHANGING ANY DEFAULT VALUE IN THE INITIALIZER, UPDATE THE PROPERTY CODE DOCUMENTATION ABOVE
```